tests/ui-testing/pages/logsPage.js

```
@@ -285,10 +293,30 @@ async logsPageURLValidation() {
+    await expect(this.page.locator('[data-test="logs-search-index-list"]')).toContainText('default, e2e_automate');
+  }
+
+  async verifyQuickModeToggle() {
+    await expect(this.page.locator(this.quickModeToggle)).toBeVisible();
+  }
+
+  async verifyQuickModeToggle() {
+
+    await expect(this.page.locator(this.quickModeToggle)).toBeVisible();
+    
+  }
```

Biome (1.9.4)

[error] Duplicate class member name "verifyQuickModeToggle"



(lint/suspicious/noDuplicateClassMembers)